metricbeat/templates/daemonset.yaml

```diff
@@ -136,6 +130,13 @@ spec:
           mountPath: /usr/share/metricbeat/{{ $path }}
           readOnly: true
           subPath: {{ $path }}
+        {{ else }}
```

Comment:
So this looked like an issue to me but I couldn't make it cause any unexpected issues when testing. After looking at [the docs](https://golang.org/pkg/text/template/) it looks like you can do a `{{ range }} {{ else }}`. I think the syntax just through me off:

> {{range pipeline}} T1 {{else}} T0 {{end}}
> 	The value of the pipeline must be an array, slice, map, or channel.
> 	If the value of the pipeline has length zero, dot is unaffected and
> 	T0 is executed; otherwise, dot is set to the successive elements
> 	of the array, slice, or map and T1 is executed.

Reply:
Yeah `{{ range }} {{ else }}` isn't documented in [Helm doc](https://helm.sh/docs/chart_template_guide/control_structures/#looping-with-the-range-action), but is fine in Go template. That said, if you think this syntax can be confusing or lead to mistake, I'm fine with replacing it by 2 entirely different `{{ range }}` loops instead.